src/components/secrets/Secret.tsx

```diff
@@ -102,19 +103,18 @@ const Secret = ({ respondOnSuccess, ...props }) => {
     return (
         <div className="form__app-compose">
             <h1 className="form__title form__title--artifacts">Secrets</h1>
-            <p className="form__subtitle form__subtitle--artifacts">A Secret is an object that contains small amount of sensitive data such as passwords, OAuth tokens, and SSH keys. 
-            <a className="learn-more__href" rel="noreferer noopener" href={DOCUMENTATION. APP_CREATE_SECRET} target="blank"> Learn more about Secrets</a></p>
+            <p className="form__subtitle form__subtitle--artifacts">A Secret is an object that contains small amount of sensitive data such as passwords, OAuth tokens, and SSH keys.
```

Comment:
small amount? i dont think this word should be used

Reply:
changed to "A Secret is an object that contains sensitive data such as passwords, OAuth tokens, and SSH keys."